### ISSUE TITLE
png-js not found

### ISSUE THREAD

**Issue by eeerrrttty** [commonsmachinery/blockhash-js#22]
dependence png-js doesnt exist. even after sudo npm install png-js and it existing on my node_modules or yarn add... Why?

**Comment by artfwo** [comment]
what do you mean by 'doesnt exist'? it is available on npm: https://www.npmjs.com/package/png-js and installs fine.

**Comment by eeerrrttty** [comment]
Yeah Artfwo, and it installs on my node_modules, which isn't recognized.

**Comment by artfwo** [comment]
But how is it not recognized? Do you get any runtime errors?

**Comment by eeerrrttty** [comment]
actually i changed to a forked version to not use these deps. It worked, ziahamza/blockhash-js

**Comment by eeerrrttty** [comment]
>  ERROR  Failed to compile with 1 errors                                            13:24:01
> 
> This dependency was not found:
> 
> * png-js in ./node_modules/blockhash/index.js
> 
> To install it, you can run: npm install --save png-js
> 

I have the png-js on my node_modules.

**Comment by eeerrrttty** [comment]
It makes completly no sense.

**Comment by eeerrrttty** [comment]
Is it a error in png-js ???

**Comment by artfwo** [comment]
What do you run to get the error above?

**Comment by eeerrrttty** [comment]
when i build my projec --> dev

**Comment by artfwo** [comment]
Ok, can you provide step by step instructions for reproducing the problem?

**Comment by eeerrrttty** [comment]
Easy. 

```
yarn add blockhash
quasar dev 
```

> "ERROR Failed to compile with 1 errors 13:24:01
> 
> This dependency was not found:
> 
> png-js in ./node_modules/blockhash/index.js"

I took a fork and fixed it then it worked, repo is on my name. (Not using PNG or JPEG dependencies)

**Comment by artfwo** [comment]
Yeah, the package published on npm is severely outdated and I no longer have the credentials to publish an update, so for now I'd suggest using the git version or your own fork.
